src/data/publications.ts

```diff
@@ -35,88 +35,2253 @@ export interface Publication {
 
 export const PUBLICATIONS: Publication[] = [
   {
-    title: 'CodeTree: A System for Learnersourcing Subgoal Hierarchies in Code Examples',
-    authors: [MEMBER.hyoungwookjin, MEMBER.juhokim, 'John Doe'],
+    title: 'ChoiceMates: Supporting Unfamiliar Online Decision-Making with Multi-Agent Conversational Interactions',
+    authors: ['Jeongeon Park', 'Bryan Min', 'Xiaojuan Ma', MEMBER.juhokim],
+    venue: 'arXiv preprint',
+    year: 2023,
+    topics: ['human-ai-interaction'],
+    type: 'preprint',
+    links: [
+      {
+        url: 'https://arxiv.org/abs/2310.01331',
+        type: PublicationLinkType.ARX,
+      },
+    ],
+  },
+  {
+    title: 'CReHate: Cross-cultural Re-annotation of English Hate Speech Dataset',
+    authors: ['Nayeon Lee', 'Chani Jung', 'Junho Myung', 'Jiho Jin', MEMBER.juhokim, 'Alice Oh'],
+    year: 2024,
+    venue: 'arXiv preprint',
+    topics: ['social'],
+    type: 'preprint',
+    links: [
+      {
+        url: 'https://arxiv.org/abs/2308.16705',
+        type: PublicationLinkType.ARX,
+      },
+    ],
+  },
+  {
+    title: 'FABRIC: Automated Scoring and Feedback Generation for Essays',
+    authors: [
+      'Jieun Han',
+      'Haneul Yoo',
+      'Junho Myung',
+      'Minsun Kim',
+      'Hyunseung Lim',
+      'Yoonsu Kim',
```

Comment:
Perhaps, we should have worked on the `members` data first...

Reply:
Plus, I think it is better to restrict the type of authors to either Members and string, not both. If we go with Members, we can keep the single source of truth policy, but we need to change the current Member class to handle external members. If we go with string, it is easier but the code will be susceptible to typos...

Reply:
Yes, I agree! I think we need to discuss this problem as well..

Reply:
I wonder if there is an API somewhere that can automatically populate these fields